backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/sales/inout/report_details_v2.jrxml

```diff
@@ -57,6 +57,9 @@ FROM 	de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details( $P{m_inout_id
 	<field name="isdiscountprinted" class="java.lang.String">
 		<property name="com.jaspersoft.studio.field.label" value="isdiscountprinted"/>
```

Comment:
pls add this change also in the other version of the report_details.jrxml

Reply:
done